client/app/styles/_base.scss

```diff
@@ -873,6 +882,7 @@ a {
     tbody {
       position: relative;
 
+
```

Comment:
check for sass-lint

Reply:
the lint issues is resolved in scss file.